### DISCUSSION TITLE
Making references compatible with Zotero plugin in word?

### DISCUSSION THREAD

**Discussion by WrongBobby** [Zettlr/Zettlr#3168] [Ideas]
I'm guessing it's quite a tall order, but I was wondering if there is any way of exporting my Zettlr project to a Word document while making the citations be in the same format as the Zotero plugin? That is, I would like to be able to edit the bibliography in Word once I have exported it (and also to be able to unlink the references). 

Other than that, I am absolutely loving Zettlr and wish I didn't have to look at Word ever again.

**Comment by nathanlesage** [comment]
Mh, there could be an option to incorporate the bibliography and the citations into an exported Word document, but I would guess that this is rather in the scope of Pandoc than with Zettlr — Zettlr provides all information to Pandoc and lets that handle all exports, but in theory it might be possible!
